src/development/ui/widgets.md

```diff
@@ -9,10 +9,21 @@ platform, and interactive widgets.
 In addition to browsing widgets by category,
 you can also see all the widgets in the [Flutter widget index](/development/ui/widgets/widgetindex).
 
-{% for section in site.data.catalog.index %}
-- ### [{{section.name}}](/development/ui/widgets/{{section.id}})
-  {:.catalog-category-title}
 
-  {{section.description}}
-{:.card}
+<div>
+{% for section in site.data.catalog.index %}
+    {% capture modulo3 %}{{ forloop.index0 | modulo:3 }}{% endcapture %}
+    {% if modulo3 == '0' %}
+        <div class="card-deck">
+    {% endif %}
+            <a class="card" href="/development/ui/widgets/{{section.id}}">
+                <div class="card-body">
+                    <header class="card-title">{{section.name}}</header>
+                    <p class="card-text">{{section.description | escape}}</p>
```

Comment:
If I don't escape here, one of the descriptions contains html (a link), which should be fine, but for some reason when it outputs it doesn't seem to be outputting correctly and it totally messes up the second card.